src/backend/gog/games.ts

```diff
@@ -888,8 +888,16 @@ class GOGGame extends Game {
   // Could be removed if gogdl handles SIGKILL and SIGTERM for us
   // which is send via AbortController
   public async stop(): Promise<void> {
+    const gameSettings = await this.getSettings()
     const pattern = isLinux ? this.appName : 'gogdl'
     killPattern(pattern)
+    if (!this.isNative())
+      await runWineCommand({
```

Comment:
would be nice to extract this `await runWineCommand(...` block into a function like killPattern so we don't have to repeat the same code in 3 different places

something like:

```ts
public async stop(): Promise<void> {
    const pattern = isLinux ? this.appName : 'gogdl'
    killPattern(pattern)
    if (!this.isNative()) {
      const gameSettings = await this.getSettings()
      shutdownWine(gameSettings)
    }
}
```

and the same for all stores, and you can define the `shutdownWine` function where `killPattern` is defined in backend utils